partiql-eval/src/eval.rs

```diff
@@ -535,6 +535,7 @@ pub enum EvalBinOp {
     Div,
     Mod,
     Exp,
+    In,
```

Comment:
nit: `In` is included under the "Arithmetic ops" section. Could include under a different heading

Reply:
Ack, will fix.